packages/components/src/components/tag/tag.stories.ts

```diff
@@ -0,0 +1,191 @@
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({ args });
+  }
+};
+
+/**
+ * The tag in all possible combinations of `selected` and `size`.
+ */
+
+export const selectedAndSize = {
+  name: 'Selected × Size',
+  parameters: { controls: { exclude: ['size', 'selected'] } },
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        x: { type: 'attribute', name: 'selected', values: ['false', 'true'] },
+        y: { type: 'attribute', name: 'size' }
+      },
+      args
+    });
+  }
+};
+
+/**
+ * The tag in all possible combinations of `removable` and `size`.
+ */
+
+export const removableAndSize = {
+  name: 'Removable × Size',
+  parameters: { controls: { exclude: ['size', 'removable'] } },
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        x: { type: 'attribute', name: 'removable', values: ['false', 'true'] },
+        y: { type: 'attribute', name: 'size' }
+      },
+      args
+    });
+  }
+};
+
+/**
+ * Use the `disabled` attribute to disable a tag. Clicks will be suppressed until the disabled state is removed.
+ *
+ * __Hint:__ If the href attribute is set i.e. the tag is rendered as a link (`<a>`),
+ * the disabled attribute is ignored, as links may not be disabled.
+ * To disable the tag in this case the href attribute has to be removed as well.
+ */
+
+export const Disabled = {
+  name: 'Disabled',
+  parameters: { controls: { exclude: ['selected', 'removable', 'disabled'] } },
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        x: { type: 'attribute', name: 'selected', values: ['false', 'true'] },
+        y: { type: 'attribute', name: 'removable', values: ['false', 'true'] }
+      },
+      constants: [
+        {
+          type: 'attribute',
+          name: 'disabled',
+          value: 'true'
+        }
+      ],
+      args
+    });
+  }
+};
+
+/**
+ * Use the `default` slot to add content to the tag.
+ * Use the `removable-indicator` slot to change the removability indicator.
+ *
+ * If you add icons to the slot, please make sure to account for accessibility by providing an alt-text.
+ */
+
+export const Slots = {
+  parameters: {
+    controls: { exclude: ['size', 'selected', 'removable', 'disabled'] }
+  },
+  render: (args: any) => {
+    return html`
+      ${['default', 'removable-indicator'].map(slot =>
+        generateTemplate({
+          axis: {
+            x: {
+              type: 'slot',
+              name: slot,
+              title: 'slot=...',
+              values: [
+                {
+                  title: slot,
+                  value:
+                    slot === 'default'
+                      ? `<slot-comp style="--slot-content: ''; --slot-height: 16px; --slot-width: 108px; font-size: 8px"></slot-comp>`
+                      : `<slot-comp slot="removable-indicator" style="--slot-content: ''; --slot-height: 16px; --slot-width: 16px; font-size: 8px"></slot-comp>`
```

Comment:
Please set slot-height to `auto` to make sure they are vertically centered. (They are not at the moment.)

![CleanShot 2023-08-30 at 09 43 13](https://github.com/solid-design-system/solid/assets/26542182/cf7e32cf-0e87-497c-a730-59b667799a65)